### DISCUSSION TITLE
start dolphinschedule-ui failed

### DISCUSSION THREAD

**Discussion by MkkOF** [apache/dolphinscheduler#11565] [Q&A]
1. with nodejs version v16.16.0
2. npm install
3. npm run dev, failed.
Build failed with 5 errors:
node_modules/.pnpm/vite@2.9.13_sass@1.53.0/node_modules/vite/dist/node/chunks/dep-80fe9c6b.js:38572:7: ERROR: [plugin: vite:dep-scan] Missing "./es/_mixins" export in "naive-ui" package
node_modules/.pnpm/vite@2.9.13_sass@1.53.0/node_modules/vite/dist/node/chunks/dep-80fe9c6b.js:38572:7: ERROR: [plugin: vite:dep-scan] Missing "./es/_mixins" export in "naive-ui" package
node_modules/.pnpm/vite@2.9.13_sass@1.53.0/node_modules/vite/dist/node/chunks/dep-80fe9c6b.js:38572:7: ERROR: [plugin: vite:dep-scan] Missing "./es/_mixins" export in "naive-ui" package
node_modules/.pnpm/vite@2.9.13_sass@1.53.0/node_modules/vite/dist/node/chunks/dep-80fe9c6b.js:38572:7: ERROR: [plugin: vite:dep-scan] Missing "./es/_mixins" export in "naive-ui" package
node_modules/.pnpm/vite@2.9.13_sass@1.53.0/node_modules/vite/dist/node/chunks/dep-80fe9c6b.js:38572:7: ERROR: [plugin: vite:dep-scan] Missing "./es/_utils" export in "naive-ui" package

how to resolve？

**Comment by luxuryone** [comment]
需要锁定下[naive-ui]版本
https://github.com/tusen-ai/naive-ui/issues/3553
